test/src/test/integration/git_helpers/remote.rs

```diff
@@ -24,6 +24,8 @@ use crate::{logging, rad::identities::create_test_project};
 
 const PASSPHRASE: &str = "123";
 
+// NOTE(finto): this test is running into issues with the key going missing part-way through the test (see https://github.com/radicle-dev/radicle-link/pull/775#issuecomment-907016375 for more commentary). I'm disabling it since we aim to have a git daemon server to replace this and the issue SHOULD NOT show up in real use scenarios, since I believe this is due to high levels of concurrency in the test suite.
```

Comment:
Maybe some newlines would help this comment.

Reply:
Yaw, I wonder why `cargo fmt` didn't do this for me :thinking: